src/main/java/seedu/address/logic/commands/FilterBuyersCommand.java

```diff
@@ -2,33 +2,42 @@
 
 import static java.util.Objects.requireNonNull;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_CHARACTERISTICS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MATCH_ALL;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_PRICE;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_PRIORITY;
 
+import java.util.function.Predicate;
+
 import seedu.address.commons.core.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.buyer.AbstractFilterBuyerPredicate;
+import seedu.address.model.buyer.Buyer;
+
 
 /**
- * Filters and lists all buyers in the buyer list that either have a price range that accepts the given price,
- * characteristics that match the given characteristics list, or have the given tag.
+ * Filters and lists all buyers in the buyer list that have a price range that accepts the given price,
+ * contains all the characteristics in the given characteristics list, or have the given the priority.
+ * More than one filtering criteria can be accepted, buyers matching any or all of the given criteria can be returned
+ * based on whether the user passes in the 'PREFIX_MATCH_ALL' flag.
  * Keyword matching is case-insensitive.
  */
 public class FilterBuyersCommand extends Command {
 
     public static final String COMMAND_WORD = "filterbuyers";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all buyers in the database who"
-            + " either have a price range that accepts the given price, characteristics that match the given"
-            + " characteristics list, or has the given tag. You must filter by exactly one criteria at a time.\n"
+            + " either has a price range that accepts the given price, contains all characteristics in the given"
+            + " characteristics list, or has the given priority. You can pass in more than one criteria at a time.\n"
+            + " Pass in " + PREFIX_MATCH_ALL + " for strict filtering, i.e. filtered buyers will satisfy"
+            + " all the conditions specified. \n"
```

Comment:
I think I found it more intuitive for the default to be filtering based on any flags. Also because if we were to filter based on all conditions by default, I can't think of an intuitive flag to change it to filter by any condition.

Reply:
Changed the default to filter based on any flag, and filtering based on all flags by passing in `-fuzzy`